dotCMS/src/main/java/com/dotcms/rest/api/v1/browsertree/BrowserTreeHelper.java

```diff
@@ -87,15 +88,13 @@ public List<Treeable> getTreeablesUnder(String siteName, User user, String uri)
     /**
      * Tries to find the select folder
      * @return Optional String, present if the folder select exists
-     * @throws DotDataException
-     * @throws DotSecurityException
      */
-    public Optional<String> findSelectedFolder (final HttpServletRequest request) throws DotDataException, DotSecurityException {
+    public Optional<String> findSelectedFolder (final HttpServletRequest request) {
 
         final BrowserAjax browserAjax = (BrowserAjax)request.getSession().getAttribute("BrowserAjax");
```

Comment:
We should not be storing the "BrowserAjax" class in session at all.  We should only store the active / open `folderId`

Reply:
Done! As we're using the Session now, this method is not necessary anymore.